kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -189,6 +189,11 @@ data:
     # [agent_settings.proxy_config]
     #    ignore_proxy_settings = "true"  # if this is not applied, default value is false
 
+    # Uses fluent-bit for perf and container inventory instead of fluentd
+    # Disables custom metrics
```

Comment:
FYI. as discussed, we will have to light up custom metrics . we can handle this work in next sprint.